docs/UserGuide.md

```diff
@@ -6,7 +6,7 @@
 
 # FindingbrUdders User Guide
```

Comment:
We should KIV that the both `Ui.png` and `findCommand.png` need to be updated so that it shows Meetings instead of saying coming in v1.4